nservicebus/msmq/distributor/deploying-to-a-cluster.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Deploying distributors to a Windows Failover Cluster
+related:
+- nservicebus/msmq/troubleshooting
+redirects:
+- nservicebus/scalability-and-ha/deploying-to-a-windows-failover-cluster
+reviewed: 2017-05-18
+---
+
+Distributors are a single-point-of-failure if deployed to just a single machine. Make them highly available on a Windows Failover Cluster to avoid this.
+
+The following procedure is a high-level guide on how to deploy distributors onto a Microsoft Windows Failover cluster. This is necessary to avoid having them as a single-point-of-failure.
+
```

Comment:
We need to explain the dependency to MSDTC. We need to mention we

* Assume *Distributed Transactions*.
* Assume a single MSMQ group with a single shared MSDTC installation

If the customer doesn't need MSDTC because they have MSMQ running in SendsAtomicWithReceive all MSDTC hints are not required.